lib/constants/constants.go

```diff
@@ -54,3 +54,16 @@ const (
 	// Ops specifies a special cloud provider - a telekube Ops Center
 	Ops = "ops"
 )
+
+// Gravity API constants redeclared here to avoid extra dependencies solely to
+// get these values
+const (
+	// SiteState come from https://github.com/gravitational/gravity/blob/7.0.0/lib/ops/constants.go#L64-L93
+	SiteStateActive   = "active"
```

Comment:
I know this is how they are defined in Gravity code, but we're trying to get away from using "Site" in our names and use "Cluster" instead. Also, same comment about Godoc format from above applies here as well.

```go
// ClusterStateActive defines the state a Gravity cluster is in when it's healthy.
ClusterStateActive = "active"
// ClusterStateDegraded defines the state a Gravity cluster is in when it's having issues.
ClusterStateDegraded = "degraded"
...
```

(feel free to tweak actual godocs)